docs/utility/donkey.md

```diff
@@ -68,9 +68,23 @@ donkey tubclean <folder containing tubs>
 ## Train the model
 **Note:** _This section only applies to version >= 4.1_
 This command trains the model.
+
 ```bash
-donkey train --tub=<tub_path> [--config=<config.py>] [--model=<model path>] [--model_type=(linear|categorical|inferred)] 
+donkey train --tub=<tub_path> [--config=<config.py>] [--model=<model path>] 
+[--model_type=(linear|categorical|inferred)] [--filter=<expr>]
 ```
+* Uses the data from the `--tub` datastore
+* Uses the config file from the `--config` path (optionally)
+* Saves the model into `--model`
+* Uses the model type `--type`
+* Supports a logic expression to filter records, for example:
```

Comment:
Yes, agreed. We can support an entry in the `myconfig.py`file, which makes it easier to avoid misspellings, etc, and it doesn't clutter the interface.